commit a59b24391bd601cc1cc6306637e2bea45e97afbc
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-10-03T17:17:41-04:00 (4 days ago)
    
    MANTA-4588 marlin "root" package.json depdencies too old to build on x86_64

diff --git a/package.json b/package.json
index 94345f5..8fc41a3 100644
--- a/package.json
+++ b/package.json
@@ -16,13 +16,13 @@
 	},
 	"dependencies": {
 		"assert-plus": "0.1.2",
-		"bunyan": "0.20.0",
-		"dtrace-provider": "0.2.8",
+		"bunyan": "1.8.12",
+		"dtrace-provider": "0.8.8",
 		"extsprintf": "1.0.1",
 		"jsprim": "0.5.0",
 		"lstream": "0.0.3",
 		"mahi": "git+https://github.com/joyent/node-mahi.git#master",
-		"moray": "3.0.0",
+		"moray": "3.7.0",
 		"node-uuid": "1.4.0",
 		"posix-getopt": "1.0.0",
 		"retry": "0.6.0",
